REQUIRE

```diff
@@ -1,4 +1,5 @@
 julia 0.6
-Deprecations
```

Comment:
This should stay in REQUIRE, since it's a dependency. We can adjust the Travis script to clone Deprecations before FemtoCleaner.

Reply:
Done,